src/picongpu/include/simulation_defines/param/radiationObserver.param

```diff
@@ -96,7 +96,7 @@ namespace picongpu
       const picongpu::float_64 phi(    my_index_phi   * delta_angle_phi   - angle_phi_start   );
 
       /* compute observation unit vector */
-      return vector_64( sinf(theta)*cosf(phi) , sinf(theta)*sinf(phi) , cosf(theta) ) ;
```

Comment:
@psychocoderHPC I fixed the use of `sinf` and `cosf`  - I am not sure why the now removed line does not show up as "outdated diff".